addons/run.sh

```diff
@@ -10,6 +10,17 @@ echo "Addon creation started"
 
 cd ${WORK_DIR}
 
+ENABLE_CLUSTER_MONITORING=${ENABLE_CLUSTER_MONITORING:-"monasca"}
+ENABLE_NODE_LOGGING=${ENABLE_NODE_LOGGING:-"true"}
+LOGGING_DESTINATION=monasca=${LOGGING_DESTINATION:-"monasca"}
+OPENSTACK_USER_ID=${OPENSTACK_USER_ID:-"ff69eeb3fd3b412a917be02ecc833963"}
```

Comment:
Do these defaults make any sense? Are these static for DevStack? Kind-of samples?